packages/resolve-scripts/src/server/express.js

```diff
@@ -139,27 +139,30 @@ const applyJwtValue = (value, res, url) => {
   const authenticationToken = jwt.sign(value, jwtSecret)
 
   res.cookie(cookieName, authenticationToken, cookieOptions)
-
   res.redirect(url || getRootableUrl('/'))
 }
 
-const bindAuthMiddleware = (route, method, middleware, options) => {
-  app[method](route, (req, res, next) =>
-    middleware(passport, options, applyJwtValue, req, res, next)
-  )
-}
-
-config.auth.strategies.forEach(strategy => {
-  const options = strategy.options
-  passport.use(strategy.init(options))
-  const routes = options.routes
-  Object.keys(routes).forEach(key => {
-    const route = getRouteByName(key, routes)
-    bindAuthMiddleware(route.path, route.method, strategy.middleware, options)
+if (
+  config.auth &&
+  config.auth.strategies &&
+  Array.isArray(config.auth.strategies)
+) {
+  config.auth.strategies.forEach(strategy => {
+    strategy.forEach(({ route, callback }) => {
+      app[route.method.toLowerCase()](

```

Comment:
Application works with incorrect 'path', but doesn't compiled with incorrect 'method' (toLowerCase error). What is the difference between them? Please, check 'method' field validation too.

Reply:
This check is correct, we support only string, we don't support object, function or others

Reply:
I mean something else. There is difference between api fields processing. Please, make validation for both of them. If you check 'path', please check the 'method' by the same way

Reply:
Maybe the best way is to not compile application and output the message about invalid 'path' or 'method' type. Because currently in case of incorrect path your application just do nothing, it's not clear for customers.